huxley/core/models.py

```diff
@@ -148,11 +148,7 @@ def update_country_preferences(self, country_ids):
     @classmethod
     def update_fees(cls, **kwargs):
         school = kwargs['instance']
-        delegate_fees = cls.DELEGATE_FEE * sum((
-            school.beginner_delegates,
-            school.intermediate_delegates,
-            school.advanced_delegates,
-        ))
+        delegate_fees = cls.DELEGATE_FEE * len(Assignment.objects.filter(school_id=school.id))
```

Comment:
You don't need to fetch all the assignments to count them. Check out the QuerySet API and see if you can find a better function. (Spoiler alert: there's totally a better function).

https://docs.djangoproject.com/en/1.6/ref/models/querysets/